src/server/request_context.cpp

```diff
@@ -215,11 +226,20 @@ std::string RequestContext::get_header(const std::string& name) const {
 }
 
 std::string RequestContext::get_user_language() const
+{
+  return userlang;
+}
+
+std::string RequestContext::get_user_language_expensive() const
```

Comment:
Should this method being called `determine_user_language` or something like that ?

`get_user_language_expensive` makes think it it the same thing that `get_user_language` but inefficient but `get_user_language` is just a "cache" for `get_user_language_expensive`

Reply:
Done